webclient/gulpfile.js

```diff
@@ -456,6 +457,26 @@ function copy_assets() {
 }
 gulp.task('assets', copy_assets);
 
+// --- Revisioning Pipeline ---
+function revision_assets(done) {
+    if (config.target === "release")
+        return gulp.src(['build/index-*.html', 'build/js/*.js'])
+                   .pipe(revAll.revision({
+                       // Currently .js only, because important css is inlined, and postloaded
+                       // css is deferred using preload, which revAll currently doesn't detect
```

Comment:
What about the content of `/assets/` i.e. `.webp`, `.svg`, `.png`, `.ico`?
Could we include the here too?
This could simplify the cache-configuration a bit in the ngnix file

Reply:
I think we could also add them to caching, too. I'll try